lago/prefix.py

```diff
@@ -1621,7 +1612,7 @@ def _deploy_host(self, host):
                 return
 
             with LogTask('Wait for ssh connectivity'):
-                if not host.ssh_reachable(tries=1):
+                if not host.ssh_reachable(tries=1, propagate_fail=False):
```

Comment:
I guess this is the change and everything is yapf?

Reply:
yep, wrong yapf version was installed while committing.